### ISSUE TITLE
[Other - Accessibility] Keyboard focus doesn't move as appropriate to workflow. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#73994]
#### VFS team name
Benefits Non-Disability

#### VFS product name
Pension 527EZ Digital Form

#### VFS feature name
Form Reactivation

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
The transition from [Other service names](https://staging.va.gov/pension/application/527EZ/military/general) to [POW Status](https://staging.va.gov/pension/application/527EZ/military/pow) seems to be doing something odd for focus management when certain conditions are met. When you have at least two other names listed and then you press Continue on Other service names, focus is visually lost and nothing is announced when I test with a screen reader. A screen reader user that doesn't get the visual cues that the page has changed may assume nothing happened or something is broken. If they press [tab] to figure out where they are, the first tab stop is the Back button --- skipping over the POW question entirely.

When I run a focus tracking script as I go from Other service names to POW Status, it says focus is going to an H3 with text "New name" (like in the list-and-loop for the other names page), which doesn't appear on the POW status page. Not sure what's happening here.

I'll note here that @gmaciolek logged #73472 as an issue detecting similar behavior on all pages when running VoiceOver on Mac. When discussing it in #accessibility-help in Slack, @jeana-adhoc identified a possible codebase change that might have caused that behavior. I'm not sure where that conversation ended up leaving. In my testing (Windows, NVDA and JAWS), I'm only encountering this one instance of that behavior.

### Link, screenshot or steps to recreate
Steps to reproduce:
1. Activate a screen reader (eg. NVDA on Windows, VoiceOver on Mac).
2. Navigate through the form to [Other service names](https://staging.va.gov/pension/application/527EZ/military/general).
3. Add two former names.
4. Press [tab] to move focus to the Continue button.
5. Press [enter].
6. Note that nothing is announced.
7. Press [tab] to move focus to the next interactive element.
8. Note that the Back button receives focus, skipping the POW Status question.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Definitely some kind of focus management bug here. Looking for focus to move to the step indicator, consistent with other steps in the workflow.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by mayacoforma** [comment]
@gmaciolek 
Hello Gretchen, I'm currently reviewing the staging/design review column in ZenHub, and I'm testing a task related to resolving a keyboard focus issue. Unfortunately, I couldn't confirm the fix with VoiceOver. Could you assist in validating it? Thanks.

**Comment by ToddWebDev** [comment]
We should be able to resolve this issue:

On stage, when you activate VoiceOver on Mac and navigate through the form to the 'Other service names' page and then add two former names, then [tab] focus to the Continue button button and press [enter] the screen reader announces the next page and shifts focus to the top of the 'POW status' page. This should resolve issue 73994.
